pkg/document/json/tree.go

```diff
@@ -223,6 +223,37 @@ func (t *Tree) Style(fromIdx, toIdx int, attributes map[string]string) bool {
 	return true
 }
 
+// RemoveStyle sets the attributes to the elements of the given range.
+func (t *Tree) RemoveStyle(fromIdx, toIdx int, attributesToRemove []string) bool {
```

Comment:
How about returning early if attributesToRemove is empty? I think this can prevent an empty operation from being created.

Reply:
I fixed `Tree.Style` and `Tree.RemoveStyle` not to make empty operation.